tiatoolbox/utils/misc.py

```diff
@@ -235,3 +237,35 @@ def mpp2objective_power(mpp):
         # Double the objective power as mpp halves
         return 10 * 2 ** (np.ceil(np.log2(0.15 / mpp)) + 2)
     raise ValueError()
+
+
+def contrast_enhancer(img, low_p=2, high_p=98):
+    """Enhancing contrast of the input RGB image using intensity adjustment.
```

Comment:
Add the method to [usage.rst](https://github.com/TIA-Lab/tiatoolbox/blob/develop/docs/usage.rst) so it is compiled on readthedocs.

Reply:
Addressed in this commit https://github.com/TIA-Lab/tiatoolbox/commit/d9752c6449023ca08f2e7a84bada8bc4130179d1